src/hooks/users/useWorldUsers.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { createContext, useContext, useMemo } from "react";
+import { isEqual } from "lodash";
+
+import { User } from "types/User";
+
+import { WithId } from "utils/id";
+import { worldUsersWithoutLocationSelector } from "utils/selectors";
+import { isDefined } from "utils/types";
+
+import { isLoaded, useFirestoreConnect } from "hooks/useFirestoreConnect";
+import { useSelector } from "hooks/useSelector";
+import { useSovereignVenueId } from "hooks/useSovereignVenueId";
+
+const noUsers: WithId<User>[] = [];
```

Comment:
```suggestion
const noUsers: never[] = [];
```

Reply:
As per https://github.com/sparkletown/sparkle/pull/1703#discussion_r666009006, nope, because we want it to have the correct type.